File: src/main/java/com/mycompany/pt/util/UsagePrinter.java
/*
 * |-------------------------------------------------
 * | Copyright © 2017 Colin But. All rights reserved.
 * |-------------------------------------------------
 */
package com.mycompany.pt.util;

import lombok.AccessLevel;
import lombok.NoArgsConstructor;
import lombok.extern.slf4j.Slf4j;

@Slf4j
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class UsagePrinter {

    public static void printUsage() {
        System.out.println("-------------------------------");
        System.out.println("java -jar PaymentTracker.jar [optional input file]");
        System.out.println("-------------------------------");
    }
}

File: kotlin/src/main/kotlin/payment/tracker/util/UsagePrinter.kt
/*
 * |-------------------------------------------------
 * | Copyright © 2019 Colin But. All rights reserved.
 * |-------------------------------------------------
 */
package payment.tracker.util

object UsagePrinter {

    fun printUsage() : Unit {
        println("------------------------")
        println("java -jar PaymentTracker.jar [optional input file]")
        println("------------------------")
    }
}
